cnc_ctrl_v1/GCode.cpp

```diff
@@ -706,22 +706,25 @@ void  G4(const String& readString){
       The G4() dwell function handles the G4 gcode which pauses for P milliseconds or S seconds.
       Only one of the two is accepted, the other ignored.
       Use maslowDelay() to remain responsive to GC during the dwell time.
+      Because maslowDelay() operates in milliseconds, round to the nearest millisecond.
+      Negative values are treated as positive (not a time machine).
```

Comment:
Do we want to open an issue for a time machine feature? 😁

Reply:
>Do we want to open an issue for a time machine feature?

only if we can make it go forward, to where it's all working just right 😁